10up/headstartwp
# Refactors to TypeScript the `wp-nextjs-ts` project

### Description of the Change

Refactoring to TypeScript of the testing `wp-nextjs-ts` project.


Closes #331

### How to test the Change

From the `/projects/wp-nextjs-ts` folder run `npm run dev` and check that the sites loads correctly and everything works.

### Changelog Entry

> Added - New example project written in TypeScript.


### Credits

Props @orballo, @nicholasio 


### Checklist:
